test/smartwallet/baseSmartWallet.test.ts

```diff
@@ -44,12 +40,8 @@ const INITIAL_SMART_WALLET_RBTC_AMOUNT = 50;
 const TOKEN_AMOUNT_TO_TRANSFER = 1;
 const RBTC_AMOUNT_TO_TRANSFER = hardhat.utils.parseEther('1');
 
-const CUSTOM_SMART_WALLET_TYPE: TypeOfWallet = 'CustomSmartWallet';
-const SMART_WALLET_TYPE: TypeOfWallet = 'SmartWallet';
-const TYPES_OF_WALLETS: TypeOfWallet[] = [
-  CUSTOM_SMART_WALLET_TYPE,
-  SMART_WALLET_TYPE,
-];
+type TypeOfWallet = 'Custom' | 'Default';
+const TYPES_OF_WALLETS: TypeOfWallet[] = ['Default', 'Custom'];
```

Comment:
Shouldn't we add here also `BoltzSmartWallet` at least?

Reply:
updated